addEvent.php

```diff
@@ -98,8 +98,123 @@
         <main class="date">
             <h2>New Appointment Form</h2>
             <form id="new-event-form" method="post">
+                <label for="name">* Name </label>
+                <input type="text" id="name" name="name" required placeholder="Enter contact name">
                 <label for="name">* Address </label>
                 <input type="text" id="address" name="address" required placeholder="Enter address">
+                <label for="name">Freezer Meals & Snacks </label>
```

Comment:
The code works as it should; at this point you are not passing the input data into a backend table, which is okay. This is just the front-end code. The code is well-organized and indented appropriately. The code seems to be efficient. I would recommend adding a comment at the top of where your changes begin to identify that this is the points program form. Where do users access access this form and where is the input data going?

Reply:
I plan to add header comments to each file I've edited as well as clarify any conflicts of where the old code is used. Users access this form within the points program module and once completed, the data should be sent to an input validation file, then input into the pointsProg table in the database.